# Request
kortex_motion_planning/JointPositions target_positions

---

#Response
bool success